configs/clustering/2020.09.16.ini

```diff
@@ -3,7 +3,7 @@ version = 2020.09.16
 description = Clustering tech grants. The clustering parameters are optimised for each tech_grants_file_name.
 
 [data]
-tech_grants_file_name = data/processed/ensemble_results.csv
+tech_grants_file_name = data/processed/ensemble/200907/ensemble_results.csv
```

Comment:
Interesting that the version of the data is inside the config file. That solves some reproducibility problems.